third_party/quaff/README.md

```diff
@@ -0,0 +1,11 @@
+# QUAFF
+
+Quaff is a Python library for manipulating, optimizing, and compiling Clifford gates.
```

Comment:
@wjhuggins PTAL at this and make sure it's accurate please

Reply:
LGTM